### ISSUE TITLE
Provide a way to comment commits other than at branch head

### ISSUE THREAD

**Issue by linquize** [google/git-appraise#31]
For example: `refs/heads/feature` -> `0123456789012345678901234567890123456789`
parent commit `01234567890123456789012345678901234567AB`
parent commit `01234567890123456789012345678901234567CD`
parent commit (`refs/heads/master`) -> `01234567890123456789012345678901234567EF`

I can only comment on `0123456789012345678901234567890123456789`.
I need to also comment on `01234567890123456789012345678901234567CD`

**Comment by ojarjur** [comment]
Yes, that should be a supported feature. We could either add a new flag to the comment command, or maybe provide some sort of interactive mode for writing comments.
